### DISCUSSION TITLE
Cashback Instantâneo

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#405] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/cashback-instantaneo/

**Comment by one-eyed-willy** [comment]
Lembrei de um [comercial antigo](<https://www.youtube.com/watch?v=sqaDd5wCgzg&t=4s>) do Guaraná Antárctica

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/cashback-instantaneo/#comment_1114002479>):

> Lembrei de um [comercial antigo](<https://www.youtube.com/watch?v=sqaDd5wCgzg&t=4s>) do Guaraná Antárctica

![null](<https://i.postimg.cc/C5CRVpX3/OIP-FXbri-u9-Xgq-Pnam0z-FW03w-Ha-E8.jpg>)
